packages/api/types.ts

```diff
@@ -244,7 +244,7 @@ type TPrivateSocketEndpoints = {
              */
             referrer?: string;
             /**
-             * Server (dxtrade and derivez).
+             * Server (dxtrade).
```

Comment:
We also need to get rid of all occurences of `pandats` like in line 254-255

service:
                | ('onfido' | 'sendbird' | 'banxa' | 'wyre' | 'dxtrade' | 'pandats' | 'ctrader')
                | ('onfido' | 'sendbird' | 'banxa' | 'wyre' | 'pandats')[];